packages/common/package.json

```diff
@@ -68,7 +72,9 @@
     "lodash.pick": "^4.4.0",
     "mime-types": "^2.1.22",
     "node-forge": "^0.7.1",
-    "p-limit": "^2.0.0",
+    "normalize-url": "5.0.0",
+	"p-limit": "^2.0.0",
```

Comment:
Weird formatting but as long as it doesn't break anything I don't care

Reply:
The local file doesn't appear to have the spacing displayed here 🤔

Reply:
3211ba096